### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    action, file, destination = command.split()
```
Comment (Grayder0152) [inline app/main.py:5]: Check that comman.split() len == 3 before it

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Please simplify code. Also make it crossplatform

File: app/main.py [hunk 2]
```diff
@@ -1 +1,21 @@
...
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3:
+        return
+    action, source_file, destination = command_parts
+    destination_dir, destination_file = os.path.split(destination)
+    if destination_file == "":
+        destination_is_dir = True
+    else:
+        destination_is_dir = False
```
Comment (ArsenPidhoretskyi) [inline app/main.py:13]: Redundant checks

File: app/main.py [hunk 3]
```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3:
+        return
+    action, source_file, destination = command_parts
```
Comment (ArsenPidhoretskyi) [inline app/main.py:8]:
```suggestion
    action, source_path, destination_path = command_parts
```

File: app/main.py [hunk 4]
```diff
@@ -1 +1,21 @@
...
+    destination_dir, destination_file = os.path.split(destination)
+    if destination_file == "":
+        destination_is_dir = True
+    else:
+        destination_is_dir = False
+    if destination_dir != "":
+        os.makedirs(destination_dir, exist_ok=True)
+    destination_path = os.path.join(destination_dir, destination_file)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:16]: Remove this also

File: app/main.py [hunk 5]
```diff
@@ -1 +1,21 @@
...
+    if os.path.exists(destination_path):
+        os.remove(destination_path)
+    os.rename(source_file, destination_path)
+    if destination_is_dir:
+        os.remove(destination_path)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:21]: Remove this check

File: app/main.py (hunk 2 above)
Comment (ArsenPidhoretskyi) [reply app/main.py:13]: Remove this part of code

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
Well done!

**Pull request closed without merging**